app/workers/scheduler/status_stat_update_scheduler.rb

```diff
@@ -74,6 +75,7 @@ def statuses_from_personalized_for_you
   # Statuses from all the 'direct follows' from all the accounts of Mammoth users
   # Take the users from Mammoth, get all the direct follows
   def statuses_from_direct_follows_for_you
+    personal_for_you = PersonalForYou.new
     mammoth_users
       .map { |acct| personal_for_you.statuses_for_direct_follows(acct) }
       .flatten
```

Comment:
Error uncovered. This need to be scoped to the method. Hoisting like JS not a thing.